src/core/core.interaction.js

```diff
@@ -264,48 +261,51 @@ export default {
 		 * @function Chart.Interaction.modes.intersect
 		 * @param {Chart} chart - the chart we are returning items from
 		 * @param {Event} e - the event we are find things at
-		 * @param {IInteractionOptions} options - options to use
-		 * @return {Object[]} Array of elements that are under the point. If none are found, an empty array is returned
+		 * @param {InteractionOptions} options - options to use
+		 * @param {boolean} [useFinalPosition] - use final element position (animation target)
+		 * @return {InteractionItem[]} Array of elements that are under the point. If none are found, an empty array is returned
 		 */
-		point: function(chart, e, options) {
+		point: function(chart, e, options, useFinalPosition) {
 			const position = getRelativePosition(e, chart);
 			const axis = options.axis || 'xy';
-			return getIntersectItems(chart, position, axis);
+			return getIntersectItems(chart, position, axis, useFinalPosition);
 		},
 
 		/**
 		 * nearest mode returns the element closest to the point
 		 * @function Chart.Interaction.modes.intersect
 		 * @param {Chart} chart - the chart we are returning items from
 		 * @param {Event} e - the event we are find things at
-		 * @param {IInteractionOptions} options - options to use
-		 * @return {Object[]} Array of elements that are under the point. If none are found, an empty array is returned
+		 * @param {InteractionOptions} options - options to use
+		 * @param {boolean} [useFinalPosition] - use final element position (animation target)
+		 * @return {InteractionItem[]} Array of elements that are under the point. If none are found, an empty array is returned
```

Comment:
all of these methods say "under", but I'm not sure that's accurate

also, nit: I probably would not capitalize the "A" to be consistent with the other comments. Actually you can probably just omit "Array of" since the type already shows it's an array